java/core/src/java/org/apache/orc/impl/WriterImpl.java

```diff
@@ -112,6 +112,7 @@ public class WriterImpl implements WriterInternal, MemoryManager.Callback {
   private long previousAllocation = -1;
   private long memoryLimit;
   private final long ROWS_PER_CHECK;
+  private final double STRIPE_SIZE_CHECK;
```

Comment:
To be consistent with `ROWS_PER_CHECK`, `STRIPE_SIZE_CHECK` -> `STRIPE_SIZE_PER_CHECK`.

Reply:
Done